src/pages/Profile/index.jsx

```diff
@@ -0,0 +1,68 @@
+import Header from 'components/Header';
+import { Link } from 'react-router-dom';
+import SProfilPage from './style';
+import apple from '../../assets/Img/apple.png';
+import samsung from '../../assets/Img/samsung.png';
+import applePay from '../../assets/Img/Apple_Pay_logo.svg';
+import cb from '../../assets/Img/logo-cb.svg';
+import larrondi from '../../assets/Img/LARRONDI.svg';
+import logo from '../../assets/Img/easyApp.png';
+import card from '../../assets/Img/easyCard.png';
+
+const solde = '37,64€';
```

Comment:
Don't forget to replace this absolute value

Reply:
yes the logic for that part isnt already implemented, that why it's static at the moment